tests/expressions_window/tests.py

```diff
@@ -1706,14 +1706,14 @@ def test_fail_insert(self):
         """Window expressions can't be used in an INSERT statement."""
         msg = (
             "Window expressions are not allowed in this query (salary=<Window: "
-            "Sum(Value(10000), order_by=OrderBy(F(pk), descending=False)) OVER ()"
+            "Sum(Value(10000)) OVER ()"
         )
         with self.assertRaisesMessage(FieldError, msg):
             Employee.objects.create(
                 name="Jameson",
                 department="Management",
                 hire_date=datetime.date(2007, 7, 1),
-                salary=Window(expression=Sum(Value(10000), order_by=F("pk").asc())),
+                salary=Window(expression=Sum(Value(10000))),
```

Comment:
Can you remind me why these changes are necessary?

Reply:
I need to remind myself as well. I don't exactly remember writing this line. I'll refresh my memory and get back to you.

Reply:
@charettes Oh, when I started validating `allow_order_by` in `Aggregate.__init__`, this test started throwing an error because `Sum` does not actually support order by. So I chose to remove the order by.

Reply:
That makes sense!